src/kpatch_log.h

```diff
@@ -3,6 +3,18 @@
 
 #include <stdio.h>
 
+#define	ERROR_SUCCESS		0	// exit code 0
+#define	ERROR_GENERAL		-255	// exit code 1
+#define	ERROR_FATAL		-254	// exit code 2
+#define	ERROR_ARGUMENTS		-253	// exit code 3
+#define	ERROR_NOPROCESS		-252	// exit code 4
+#define	ERROR_NOPATCH		-251	// exit code 5
+#define	ERROR_PATCH		-250	// exit code 6
+#define	ERROR_UNPATCH		-249	// exit code 7
+#define	ERROR_PATCH_UNPATCH	-248	// exit code 8
```

Comment:
This name is ambiguous. Please provide comments for all the error codes as shown above. We will later include these into man page.